src/model.ts

```diff
@@ -585,7 +585,7 @@ export class Model extends EventBus<any> implements CommandDispatcher {
    * This prove to be necessary if the client did not trigger that evaluation in the first place
    * (e.g. open a document with several sheet and click on download before visiting each sheet)
    */
-  exportXLSX(): XLSXExport {
+  async exportXLSX(): Promise<XLSXExport> {
```

Comment:
as discussed: don't support old databases. There's only a handful of them (if there are some)